ansible/roles/riakcs/join_cluster.yml

```diff
@@ -15,7 +15,7 @@
 
 #- debug: var=riak_cluster_status
 
-- set_fact: riak_control_node_name="{{ hostvars[riakcs_control]['riak_nodename'] }}"
+- set_fact: riak_control_node_name="{{ riak_nodename }}"
```

Comment:
Oh! @esoergel that makes a lot of sense after doing our research. Somehow this ended up working? Maybe we deployed just that one time off of master instead of this branch? @millerdev I think at some point, running this wasn't working and that's why Ethan and I looked into it and ended up making this change.